src/core/public/utils/integration_tests/__fixtures__/frozen_object_mutation/index.ts

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-import { deepFreeze } from '../../../deep_freeze';
+import { deepFreeze } from '../../../../../utils/deep_freeze';
```

Comment:
The other imports for `deepFreeze` pull this directly off of `utils` instead of `utils/deep_freeze`; did you want to do that here as well?

Reply:
this integration test compiles TS and declares its own tsconfig https://github.com/restrry/kibana/blob/5128c9bf3d2fc508d95d27414b912e1d12be5a55/src/core/public/utils/integration_tests/deep_freeze.test.ts
if I import `../../../../../utils/` TS compilation fails due to different setup for the Kibana project and test. I don't think we should keep both in sync, so I just narrowed import to the specific file.